packages/kbn-shared-ux-components/src/page_template/solution_nav/solution_nav.scss

```diff
@@ -18,6 +18,9 @@ $euiSideNavEmphasizedBackgroundColor: transparentize($euiColorLightShade, .7);
   .kbnPageTemplateSolutionNav__avatar {
     margin-right: $euiSize;
   }
+  
+  display: flex;
+  flex-direction: column;
```

Comment:
Can you move this up to before the breakpoint mixin (line 13) otherwise it's hard to follow what styles are being applied directly to the selector.